XCUITests/AuthenticationTest.swift

```diff
@@ -34,8 +40,10 @@ class AuthenticationTest: BaseTestCase {
 
     // Smoketest
     func testPromptPassCodeUponReentry() {
-        navigator.goto(URLBarOpen)
-        navigator.back()
+        waitForExistence(app.textFields["url"], timeout: 5)
+        waitForExistence(app.buttons["Go"], timeout: 5)
+        navigator.performAction(Action.CloseURLBarOpen)        
+        navigator.nowAt(NewTabScreen)
         navigator.performAction(Action.SetPasscode)
```

Comment:
@isabelrios I did the requested changed. I had to add two extra lines. This was the only way the test passed 5 out of 5 times. Without `waitForExistence(app.textFields["url"], timeout: 5)` and `waitForExistence(app.buttons["Go"], timeout: 5)` the test failed intermittently.

Reply:
Just realized that may fail... since the Go key in Bitrise is "go".. see screenshot in the logs
![Screenshot_4CF167F1-3037-4772-BEA1-FA4AFB242A0D](https://user-images.githubusercontent.com/1897507/114174940-a7a92d80-9939-11eb-83f7-ec6192951910.jpg)

Reply:
Changed to `waitForExistence(app.buttons["urlBar-cancel"], timeout: 5)`